### PR TITLE
Student Dashboard - My Questions

### PR DESCRIPTION
**Pull request by realARCHaim**
https://trello.com/c/Hr7E9Ls4

### CODE REVIEW

File: askup/static/assets/application.css [hunk 1]
```diff
@@ -9365,3 +9369,56 @@ select.form-control, input.form-control {
...
+    padding: 5px;
+    line-height: 32px;
+}
+
+.subject-name-text {
+    cursor: default;
+    float: left;
+    font-size: 24px;
+    width: 100%;
+    word-wrap: break-word
```
Comment (idegtiarov) [inline askup/static/assets/application.css]: add semicolon

File: askup/utils/views.py [hunk 2]
```diff
@@ -284,3 +289,36 @@ def apply_filter_to_queryset(request, filter, queryset):
...
+    return subjects
+
+
+def compose_user_qsets(queryset):
+    """
+    Compose user qsets list from the questions queryset.
+    """
+    qsets = []
+
+    for question in queryset:
```
Comment (idegtiarov) [inline askup/utils/views.py]: This function could be avoided with the using `values_list()` in the function `get_user_subjects`.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: askup/static/assets/application.css (hunk 1 above)
Comment (realARCHaim) [reply askup/static/assets/application.css]: done

File: askup/utils/views.py (hunk 2 above)
Comment (realARCHaim) [reply askup/utils/views.py]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**